### DISCUSSION TITLE
请问如何获取带?page=2后缀的网站内容

### DISCUSSION THREAD

**Discussion by bug0-1** [DIYgod/RSSHub#12027] [Q&A] [answered]
### 这是一个什么样的功能？

有些网址的页码后缀为?page=2，而默认路由获取的仅有第一页的内容，如何更改路由以获得后面页数的内容

### 这个功能可以解决什么问题？

获取不同页码的内容

### 这不是重复的功能请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保这项功能尚未被请求。

**Comment by TonyRL** [comment, marked as answer]
> 很多路由上限10条，无法增大。
> 
> 1. 消耗大量服务器资源。
> 2. 容易被反爬虫。
> 3. 绝大部分过时 feed 没有意义。

_Originally posted by HenryQW in https://github.com/DIYgod/RSSHub/issues/572#issuecomment-416757360_

> RSS 是用来获取更新，增大条目无意义
> 
> 特殊需求请在相应模块里自行修改

_Originally posted by DIYgod in https://github.com/DIYgod/RSSHub/issues/572#issuecomment-416802920_

> rss 就是用来跟踪内容源的更新内容的，采集全部内容不是rsshub要做的，建议你去找一个爬虫

_Originally posted by yefoenix in https://github.com/DIYgod/RSSHub/issues/7291#issuecomment-809019525_
